internal/cli/atlas/dbusers/create.go

```diff
@@ -33,12 +33,13 @@ import (
 
 type CreateOpts struct {
 	cli.GlobalOpts
-	username   string
-	password   string
-	x509Type   string
-	awsIamType string
-	roles      []string
-	store      store.DatabaseUserCreator
+	username        string
+	password        string
+	x509Type        string
+	awsIamType      string
+	deleteAfterDate string
```

Comment:
[nit] we can make this just `deleteAfter` no need for more (for reference we have it in the client because that's the name in the API but we don't need to fall into the errors of the API